runtime/compiler/control/rossa.cpp

```diff
@@ -1820,11 +1863,27 @@ aboutToBootstrap(J9JavaVM * javaVM, J9JITConfig * jitConfig)
          {
          javaVM->sharedClassConfig->runtimeFlags &= ~J9SHR_RUNTIMEFLAG_ENABLE_AOT;
          TR_J9SharedCache::setSharedCacheDisabledReason(TR_J9SharedCache::AOT_DISABLED);
+#if defined(JITSERVER_SUPPORT)
+         if (compInfo->getPersistentInfo()->getRemoteCompilationMode() == JITServer::SERVER)
+            {
+            // TODO: Format the error message to use j9nls_printf
+            fprintf(stderr, "Aborting Compilation: SCC/AOT must be enabled and can be stored in JITServer Server Mode.");
```

Comment:
The message is not clear at all here. We are not aborting a compilation, but the entire JVM. Maybe something like: "Error: -Xaot:nostore option is not compatible with JITServer mode" is better.